dist/performance.js

```diff
@@ -171,24 +160,70 @@ function getGamingMetrics(rawDoc) {
                     styleObj['top'] == '0px' &&
                     styleObj['left'] == '0px' &&
                     styleObj['z-index'] == '999'){
-                returnObj['fidIframeOverlay'] = true;
+                returnObj['fidIframeOverlayStrict'] = true;
             }
         }
+
+        let cover90viewport = false;
+        doesElementCoverPercentageOfViewport(iframeElement, 90)
+        returnObj['fidIframeOverlaySoft'] = cover90viewport && styleObj['z-index'] === findHighestZ(rawDoc);
```

Comment:
Since this is the "soft" version, WDYT about any positive z-index to be indicative that the intent was to overlay the iframe? Your way is more like the strict interpretation even if not the literal z-index used by the blog post.

Reply:
@mel-ada I think we should not use the z-index check in the soft version.   someone can achieve an iframe overlay without a z-index by properly positioning the iframe & elements.  I have added it earlier because I found some iframe overlay examples were using that property